main/guides/integration/chain-integration.md

```diff
@@ -36,15 +36,15 @@ agd version --long
 # FAQ
 
 - How are transactions encoded?
-https://docs.cosmos.network/v0.45/core/encoding.html
+[https://docs.cosmos.network/v0.45/core/encoding.html](https://docs.cosmos.network/v0.45/core/encoding.html)
 - What data is needed to create a transaction (last block hash, nonce, sender public key, etc.)?
 https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation
 - What data is signed and how is that data obtained (for example truncated SHA256 of transaction data)?
-https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation
+[https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation](https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation)
```

Comment:
Let's avoid urls-in-your-face, please:

```suggestion
[Transaction Generation](https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation)
```

and likewise throughout.

Reply:
[eb9064d](https://github.com/Agoric/documentation/pull/970/commits/eb9064dd85a1bf658e75a44a76b6d336335d30f3) (updated PR body with new After)